src/lib/utils/string.ts

```diff
@@ -13,3 +15,10 @@ export const truncateString = (
 	str.length > len
 		? str.slice(0, len > tail.length ? len - tail.length : len) + tail
 		: str;
+
+/**
+ * Capitalize string
+ * @param str to capitalize
+ * @returns capitalized string
+ */
+export const capitalizeString = (str: string) => capitalize(str);
```

Comment:
Is it worth re-defining/re-exporting the lodash `capitalize` method here? I think it makes sense to do this when you're expanding an external library function, or when you're not 100% sold on the library function and would like to easily swap it out at some point. In this case, `capitalize` is a very standard function and won't be extended or replaced in the future - may be worth just using `capitalize` straight from lodash to reduce code base bloat.